### PR TITLE
solved task

### PR DESCRIPTION
**Pull request by hulchak**
https://hulchak.github.io/react_person/

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
```
Comment (VitaliyBondarenko1982) [inline src/components/Person/Person.jsx:3]: suggest destructure person

File: src/components/Person/Person.jsx [hunk 2]
```diff
@@ -1 +1,20 @@
...
+      </>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my `}
+        {person.sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```
Comment (VitaliyBondarenko1982) [inline src/components/Person/Person.jsx:18]: we just need to decide which text will be rendered inside Person__partner element.
and better to make two separate ternary expressions in component for make code more readable.
```
const partner = person.sex === 'm' ? 'wife' : 'husband';
const personPartner = person.isMarried 
  ? `${person.partnerName} is my ${partner}`
  :  'I am not married';
```

```suggestion
      <p className="Person__partner">
        {personPartner}
      </p>
```

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Good job.
Please check comments.

**Review by nazarmatsevych — APPROVED** [review]
🔥

**Pull request closed without merging**